schematic/models/README.md

```diff
@@ -52,13 +52,13 @@ Generate a manifest based on a schema.org schema, which in this case is specifie
 
 ```python
 print("Testing manifest generation based on an additionally provided JSON schema..")
-HTAPP_VALIDATION_SCHEMA = os.path.join(DATA_PATH, model["demo"]["htapp_validation_file_location"])
+HTAPP_VALIDATION_SCHEMA = CONFIG["model"]["demo"]["htapp_validation_file_location"]
```

Comment:
Hmm, is this still in the config (i.e. 'demo' section is not part of the new config?)?
Might want to edit/update the readme to reflect latest config.

Reply:
@milen-sage I was thinking we could go ahead and remove the README's (and maybe even the `examples/` folder within the sub-packages?) altogether, because as @BrunoGrandePhD pointed out it's becoming tedious to keep these README's in sync with the sub-packages. And they were created initially before we moved the `examples/` directory to the top-level.